examples/userapi/07_functions_on_subdomains.ipynb

```diff
@@ -0,0 +1,3028 @@
+{
```

Comment:
<p>NIT: define devito's imports on top, actually you can follow the convention used in the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I always use the following and it seems to be convention throughout the codebase:</p>

<pre class="ql-syntax" spellcheck="false">import sysfoo
...
import foo as bar
...
from sysfoo import bar
...
from foo import bar
...
from devito import bar
...
from examples import bar
</pre>